addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -83,12 +83,37 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
     // Private
     //--------------------------------------------------------------------------
 
+    _getAttachmentsDomain: function () {
+        var domain = ['|', ['id', 'in', _.pluck(this.attachments, 'id')]];
+        if (this.recordData.model) {
+            domain = domain.concat([
+                '&',
+                '&',
+                ['res_model', '=', this.model],
+                ['res_id', '=', this.res_id|0],
+                ['create_uid', '=', session.uid]]);
+        } else {
+            domain = domain.concat([
+                '&',
+                ['res_model', '=', this.model],
+                ['res_id', '=', this.res_id|0]]);
+        }
+        if (this.recordData.model) {
+            domain = ['|'].concat(domain).concat([
+                '&',
+                ['res_model', '=', this.recordData.model],
+                ['res_id', '=', this.recordData.res_id|0]]);
+        }
+        return domain;
```

Comment:
Mmh, don't understand the heuristic here and even if it is correct, I think this method could be written in 3 times less lines.

Reply:
mmh, yes maybe, I'll look for this